actions/java-gradle-build/README.md

```diff
@@ -0,0 +1,29 @@
+# java-gradle-build
+
+This action builds Java artifacts using Gradle and uploads `.jar` files as a pipeline artifact.
```

Comment:
What is exactly a `pipeline artifact`? I would only use `artifact(s)` here.

Reply:
I tried to find a name that describes an artifact that can be shared between pipeline jobs by uploading it with this action: https://github.com/actions/upload-artifact. I think just calling it `artifact` is to generic. Do you have a better name suggestion?